wiki/Beatmap/Overall_difficulty/en.md

```diff
@@ -56,7 +60,7 @@ If the player hits outside of the 50's hit window, it will count as a miss. In c
 
 ## Sliders and spinners
 
-In [osu!](/wiki/Game_mode/osu!), sliders will reward a 300 as long as they are hit within the 50's hit window. This is sometimes referred to as *slideracc* and is removed in [ScoreV2](/wiki/Game_modifier/ScoreV2)
+In [osu!](/wiki/Game_mode/osu!), [sliders](/wiki/Gameplay/Hit_object/Slider) will reward a 300 as long as they are hit within the 50's hit window. This is sometimes referred to as *slideracc* and is removed in [ScoreV2](/wiki/Game_modifier/ScoreV2)
```

Comment:
I know this is in the original, but I believe the whole sentence about "slideracc" is back to front (it was *added* in ScoreV2 / lazer, not removed). Also I think "slideracc" should be replaced with "slider accuracy" as there are very few references to the weird abbreviation on google.

Reply:
this was probably meant to be "slider leniency" instead, but it can go both ways